### ISSUE TITLE
media-gfx/blender-2.81.ebuild: openvdb_api must be set

### ISSUE THREAD

**Issue by gerion0** [redchillipadi/ebuild-overlay#2]
Either use `openvdb_api_4` or `openvdb_api_5` must be set currently. I guess, this is because of a missing question mark:
```
-       openvdb ( || ( openvdb_abi_4 openvdb_abi_5 ) )
+       openvdb? ( || ( openvdb_abi_4 openvdb_abi_5 ) )
```

**Comment by gerion0** [comment]
Problem persists with blender-2.81 ebuild.

**Comment by redchillipadi** [comment]
Thanks, fixed by commit fb4997d8522eb0d41289199dbe53cf4e7f16de63
